third_party/docfx/templates/devsite/partials/uref/class.header.tmpl.partial

```diff
@@ -1,12 +1,51 @@
 {{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
 
 {{>partials/disclaimer}}
+{{#syntax.content.0}}
+<div class="codewrapper">
+  <pre class="prettyprint{{#wrapdeclarationcode}} wrap-code{{/wrapdeclarationcode}}"><code>{{syntax.content.0.value}}</code></pre>
+</div>
+{{/syntax.content.0}}
 {{#summary}}
 <div class="markdown level0 summary">{{{summary}}}</div>
 {{/summary}}
 {{#conceptual}}
 <div class="markdown level0 conceptual">{{{conceptual}}}</div>
 {{/conceptual}}
+{{#syntax.parameters.0}}

```

Comment:
I don't see any golden updates from this parameters and returns addition. Do we need it?

Reply:
Updated the goldens to include ones with parameters. `bigquery_stroage` doesn't seem to have any docstrings on returns from constructors.